test-requirements-34.txt

```
@@ -1,3 +1,3 @@
-numpy < 1.19.0
+numpy<1.17.0
```

Probably rather <1.16 – that seems to be the version where they ended Py3.4 support.
```suggestion
numpy<1.16.0
```
Seems to be a general issue, though, and doesn't belong into a PR to make Windows builds work.

---

Idk, my tests with py3.4 passed the numpy install stage, so 17 will work too

---

It probably works, but there aren't any wheels for 3.4 any more.

---

Well, okay, let's say this is so :)